CHANGELOG.md

```diff
@@ -8,13 +8,12 @@
 * `DistanceMatrix.from_iterable` now accepts a `validate=True` parameter. ([#1343](https://github.com/biocore/scikit-bio/issues/1343)).
 * Added `IntervalMetadata` and `Interval` classes in `skbio.metadata` to store, query, and manipulate information of a sub-region of a sequence. ([#1414](https://github.com/biocore/scikit-bio/issues/1414)).
 * `Sequence` and its child classes (including `GrammaredSequence`, `RNA`, `DNA`, `Protein`) now takes `IntervalMetadata` in their contructor API. Some of their relevant methods are also updated acoordingly. ([#1430](https://github.com/biocore/scikit-bio/pull/1430))
+* GenBank parser now reads and writes `Sequence` or its subclass objects with `IntervalMetadata`. ([#1440](https://github.com/biocore/scikit-bio/pull/1440))
```

Comment:
It'd be nice to note the performance gains from this change (they are huge) and reference the original issue that prompted the creation of interval metadata (#1159). Probably makes sense in the `Performance enhancements` section below.

There were also some backward-incompatible changes made here (for example, `positional_metadata` is no longer used in favor of `interval_metadata`, and the keys used in `metadata`/`positional_metadata` have changed). Can you note all backward-incompatible changes under `Backward-incompatible changes [experimental]` section? Basically anything that could break existing user code relying on the GenBank reader/writer.